src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -46,7 +46,10 @@ protected function enqueueResources(Group $pools, Database $dbForConsole): void
             }
 
             $delay = $scheduledAt->getTimestamp() - (new \DateTime())->getTimestamp();
-
+            $schedule['data'] = $dbForConsole->getDocument(
```

Comment:
Let's move it 1 line above, it will ensure better precision of delay

Reply:
Done.